config/pod.yaml.hbs

```diff
@@ -146,6 +150,12 @@ spec:
       hostPath:
         path: {{cache.path}}/events/{{pipeline_id}}/{{event_id}}
     {{/if}}
+    {{#if cache.dlcEnabled}}
+     - name: sd-docker-cache
+      type: DirectoryOrCreate
+      hostPath:
+        path: {{cache.dlc_path}}/pipelines/{{pipeline_id}}/{{cache.dlc_mount_path}}
```

Comment:
shouldn't this be under `{{cache.path}}` ?

Reply:
keeping it separate from cache.path, just in case we intend to have a different value